### ISSUE TITLE
Checking Credits have all relevant folks before publication

### ISSUE THREAD

**Issue by terracoda** [phetsims/a11y-research#134]
With the recent wonderful publications of _Resistance in a Wire_ and _Friction_, we need to ensure that we have a good process for checking the credits before the sim is actually published.

There is a lot of people to thank, so best to make sure everyone is there including accessibility folks and sound designers. 

- _Resistance in a Wire_ seems to be missing a couple of relevant A11y folks in the Team list, specifically Emily Moore and Taliesin Smith. The Sound designers are also missing. I am not sure who the sound designers are.
- _Friction_ is missing at least Taliesin Smith from the Team list.

More generally, we need to decide as a team if/when we need to recognize contributions from the A11y folks in the Team list for sims that contain A11y features like alternative input but not full description or sound - a sim like _Coulomb's Law_, for example.

I am tagging @ariel-phet, @emily-phet, @jessegreenberg, @zepumph and @jbphet.

I'll assign to @zepumph, but please re-assign if appropriate.

I am adding the **project:mastercheck-list label,** too. I think checking the Credits is on the master-checklist, but maybe the item needs more detail for clarity now that we are publishing sims with multiple features and diverse collaborations.

@emily-phet could you add who the sound designers are for RIAW.
@ariel-phet could you add priority level please.

**Comment by jbphet** [comment]
Once the additional names have been decided upon for Friction and RIAW, please log an individual issue to update the credits for each.  I should be assigned the RIAW issue and @zepumph should get the Friction one (I think).

**Comment by ariel-phet** [comment]
@terracoda probably we need to develop a new checklist. The master checklist is for the initial publication of a sim, and really doesn't get looked at after that initial publication (usually the issue is closed).

It might be good to have a "major new features" or such checklist for this kind of work when descriptions/sonification or such are added.

Checklists have served us fairly well, since it is easy for a busy developer (like @zepumph!) to be focused on trying to get the sim out and miss thinking about something like updating credits.

Perhaps you and @emily-phet could put together a draft of a checklist targeted at these kinds of projects since you have the most experience, and then we can refine it in a dev meeting. We can then plan to paste that checklist into an issue when such work begins (much like we paste in the master checklist as the first issue when a new sim repo is created). As far as who should go in the credits list, I am fairly ambivalent (as in I am fine adding whoever you and Emily think is appropriate).

I will go ahead and make a clarification in the master checklist regarding diverse collaborators when it comes to credits, so when newer sims are made "from scratch" that include these features we are ensuring people are getting credit.

**Comment by emily-phet** [comment]
@ariel-phet - I think this conversation may need to start in dev meeting. @terracoda and I are not directly involved with publishing the sim, so it will be difficult for us to create a checklist that aligns with actual development processes.

Regarding credits, the primary issue is **where** people should be listed. For example, as sims get description etc (lots and lots of effort across multiple people) it makes sense to me for those involved to be added to the "team" list. For a sim with alternative input only, it seems like it should be discussed if a the "team" list is then the appropriate place.

**Comment by terracoda** [comment]
@emily-phet, I will at least add a check credits item to the description check list that is under construction and in use for GFL and GFLB (and wasn't yet created for RIAW and Friction).

That will address at least part of the issue, making sure the designers for description and/or sound are added.

@emily-phet, as for the sound designers for RIAW is it both Mike Winters and Ashton Morris, anyone else?

**Comment by terracoda** [comment]
As per @jbphet's suggestion in https://github.com/phetsims/a11y-research/issues/134#issuecomment-467948060 I posted https://github.com/phetsims/friction/issues/176 and https://github.com/phetsims/resistance-in-a-wire/issues/208

**Comment by emily-phet** [comment]
Also note - each sim page has a 'credits' section at the bottom. It doesn't look like these have been updated in a really long time.

**Comment by terracoda** [comment]
@emily-phet and @ariel-phet, I added a Check/verify credits to our recent Description checklist for Gravity Force Lab (https://github.com/phetsims/gravity-force-lab/issues/109) and Gravity Force Lab Basics (https://github.com/phetsims/gravity-force-lab-basics/issues/84).

Our Description Check List is still evolving. I have not yet made a template for it. That will come soon.

@emily-phet, my understanding of the credits section on the sim page was for original sim design only, but I could be wrong.

@emily-phet, I am comfortable closing this issue if you are.

**Comment by terracoda** [comment]
@ariel-phet, I'll put the priority 3 label on  https://github.com/phetsims/friction/issues/176 and https://github.com/phetsims/resistance-in-a-wire/issues/208

**Comment by terracoda** [comment]
@emily-phet, can we close this issue?
